src/latexify/transformers/transformer_utils.py

```diff
@@ -0,0 +1,30 @@
+"""Utilites for latexify.frontend."""
```

Comment:
will add tests to test that appropriate transformer is called if you think this is an ok approach

Reply:
I'm not sure what "if you think this is an ok approach" mean. what is intended here?